emcc.py

```diff
@@ -768,6 +768,9 @@ def get_file_suffix(filename):
   def optimizing(opts):
     return '-O0' not in opts
 
+  def need_llvm_debug_info(options):
+    return options.debug_level >= 3 or shared.Settings.CYBERDWARF
```

Comment:
From `emcc --help`, LLVM debug information should be only included when building with `-g4`, not `-g3`, so I'm not sure if this is an intentional change.
```
      * "-g3": Preserve variable names (this is the same as -g).

              function _addAndPrint($left, $right) {
                      $left = $left | 0;
                      $right = $right | 0;
                      _printAnInteger($left + $right | 0);
              }

           Note: Variable names in the output will not necessarily
             match the original variable names in source. They are,
             however, usually similar enough to infer the purpose of
             the variables.

      * "-g4": Preserve LLVM debug information. This is the highest
        level of debuggability. If "-g" was used when compiling the
        C/C++ sources, this shows line number debug comments, and
        generates source maps.

           Note:

             * This debugging level may make compilation at
               optimization level -O1 and above significantly slower,
               because JavaScript optimization will be limited to one
               core (the default in "-O0").

             * Source maps allow you to view and debug the *C/C++
               source code* in your browser's debugger! This works in
               Firefox, Chrome and Safari.
```

Reply:
Thanks @PiotrSikora, looks like those docs are now out of date, I'll fix that!